src/neuroconv/datainterfaces/__init__.py

```diff
@@ -132,3 +132,22 @@
     CsvTimeIntervalsInterface,
     ExcelTimeIntervalsInterface,
 ]
+
+interface_lookup = {interface.__name__: interface for interface in interface_list}
+ecephys_recording_interfaces = {
```

Comment:
Would it be possible to export a single dictionary here (`interface_categories`) that has these variables as keys without the _interface suffix? This would be how I'd design it myself—though I'm not sure of any tradeoffs that come in Python

Reply:
Are you thinking a double-nested structure?

```
{
    ecephys_recordings: {"SpikeGLX": SpikeGLXRecordingInterface, ...},
    icephys_recordings: {"Abf": AbfInterface},
    behavior: {"Video": VideoInterface, ...},
    ....
}
```

Also what do you think of the approach where the "<modality>Interface" is stripped?

Reply:
At least for NWB GUIDE, the idea would be to send all names with associated categories to the client. Then receive a list of chosen names that can be accessed with the `interfaces` dict and processed using another function.

Reply:
Can you give an example of what the GUIDE would request, as you're imagining it right now?

Reply:
Double-nested would be what I'd go with. Also apologies for the hodge-podge of messages, I'd assumed that Github Issues would populate with new responses without a refresh...

Reply:
No worries, this is going great! I'll get that double nested ready in a second